tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -102,9 +102,9 @@ def test_fail_fast_on_incompatible_brokers(self, broker_version):
 
         processor.node.account.ssh(processor.start_cmd(processor.node))
         with processor.node.account.monitor_log(processor.STDERR_FILE) as monitor:
-            monitor.wait_until('FATAL: An unexpected exception org.apache.kafka.streams.errors.StreamsException: Could not create internal topics.',
+            monitor.wait_until('FATAL: An unexpected exception org.apache.kafka.streams.errors.StreamsException: Could not create topic kafka-streams-system-test-broker-compatibility-KSTREAM-AGGREGATE-STATE-STORE-0000000001-changelog.',
```

Comment:
Quick note, we just observed one of the system tests `streams_broker_compatibility_test.py` that did not print the message from stdout, but from the main thread,  the bulk of the message is the same, but the start of the message does not include `FATAL`